amm/interp/src/main/scala/ammonite/runtime/Interpreter.scala

```diff
@@ -699,9 +700,15 @@ object Interpreter{
     Name(wrapperName.raw + (if (wrapperIndex == 1) "" else "_" + wrapperIndex))
   }
 
-  def initPrinters(output: OutputStream, error: OutputStream) = {
+  def initPrinters(output: OutputStream, error: OutputStream, verboseOutput: Boolean) = {
     val colors = Ref[Colors](Colors.Default)
-    val printStream = new PrintStream(output, true)
+    val printStream = verboseOutput match{
+      case true => new PrintStream(output, true)
+      //make a dummy printstream in case of non-verbose output
+      case false => new PrintStream(new OutputStream(){
+        def write(i: Int) = ()
+      })
+    }
```

Comment:
Rather than rewiring `printer.out`, could we instead put an if-statement around the `Compiling...` statement? This would cause some funny behavior when in the REPL (e.g. stuff wouldn't get echoed back properly after running a statement) whereas if we put the if around `Compiling...`, the only change would be that we don't see `Compiling...` when loading a script from the Ammonite REPL, which is probably less surprising